test/externalTests.sh

```diff
@@ -41,6 +41,11 @@ DIR=$(mktemp -d)
     echo "Running Zeppelin tests..."
     git clone --depth 1 https://github.com/OpenZeppelin/zeppelin-solidity.git "$DIR"
     cd "$DIR"
+    # Fix some things that look like bugs
+    sed -i -e 's/let token = await ERC827TokenMock.new();/let token = await ERC827TokenMock.new(accounts[0], 0)/;' test/token/ERC827/ERC827Token.js
+    sed -i -e 's/CappedCrowdsale.new(this.startTime, this.endTime, rate, wallet, 0)/CappedCrowdsale.new(this.startTime, this.endTime, rate, wallet, 0, this.token.address)/' test/crowdsale/CappedCrowdsale.test.js
+    sed -i -e 's/RefundableCrowdsale.new(this.startTime, this.endTime, rate, wallet, 0, { from: owner })/RefundableCrowdsale.new(this.startTime, this.endTime, rate, wallet, 0, this.token.address, { from: owner })/' test/crowdsale/RefundableCrowdsale.test.js
```

Comment:
Checked in the source, these two look reasonable. Strange though it doesn't fail on node 8?!

Reply:
Does web3.js behave differently depending on node version?

Reply:
I have NO idea...